libs/cli/src/generators/ui/libs/ui-accordion-helm/files/lib/hlm-accordion-trigger.directive.ts.template

```diff
@@ -10,7 +10,7 @@ import { ClassValue } from 'clsx';
 		'[style.--tw-ring-offset-shadow]': '"0 0 #000"',
 		'[class]': '_computedClass()',
 	},
-	hostDirectives: [BrnAccordionTriggerDirective],
+	hostDirectives: [BrnAccordionTriggerComponent],
```

Comment:
Question: Is this allowed in Angular? Using a component as a directive?

Reply:
No and good catch! I will need to remove that. Got carried away with find and replace haha